java-client/src/main/java/co/elastic/clients/elasticsearch/_helpers/bulk/IngesterOperation.java

```diff
@@ -154,6 +154,16 @@ private static void copyBaseProperties(BulkOperationBase op, BulkOperationBase.A
             .versionType(op.versionType());
     }
 
+    private static void copyIndexProperties(IndexOperation<?> op, IndexOperation.Builder<?> builder) {
+        copyBaseProperties(op, builder);
+        builder.pipeline(op.pipeline());
```

Comment:
Can you please add `requireAlias` that is also part of index operations?

```suggestion
        builder.pipeline(op.pipeline());
        builder.requireAlias(op.requireAlias());
```

Reply:
Yes sure, it is done.